main.py

```diff
@@ -344,53 +387,95 @@ def __init__(self, *args):
 		self.inst = Xqemu()
 		self.settings = SettingsManager()
 		self.settings.load()
-		self.runButton.setText('Start')
-		self.pauseButton.setText('Pause')
-		self.screenshotButton.setText('Screenshot')
+		self.fileformats = [
+			{
+				'name': 'XISO',
+				'offset': [
+					0x10000
+				],
+				'signature': [
+					'MICROSOFT*XBOX*MEDIA'
+				]
+			},
+			{
+				'name': 'ISO9660 CD/DVD image file',
+				'offset': [
+					0x9001,
+					0x8801,
+					0x8001
+				],
+				'signature': [
+					'CD001'
+				]
+			}
+		]
```

Comment:
I think you can hard-code this (in a separate function). It's unlikely we'll add more formats, and even if we do, it's only like 5 lines per format.

We might also want to perform additional checks for some formats, and adding this here would be.. complicated and weird?